docs/android/upgrading.md

```diff
@@ -4,21 +4,36 @@
 
 Version 7 of the Embrace Android SDK contains the following breaking changes:
 
+- The `embrace-android-fcm` and `embrace-android-compose` gradle modules are no longer automatically added by Embrace's gradle plugin. Add them as dependencies in your `build.gradle` file if you require them.
 - The `startMoment/endMoment` API has been removed. Use `startSpan/recordSpan` instead.
 - `Embrace.AppFramework` is now its own top level class, `AppFramework`
 - `Embrace.LastRunEndState` is now its own top level class, `LastRunEndState`
-- Several public APIs are now implemented in Kotlin rather than Java. Generally this will not affect backwards compatibility but the following may have slight changes to their signatures:
-  - `EmbraceNetworkRequest` Java overloads replaced with default parameters
-- View taps do not capture coordinates by default. Set `sdk_config.taps.capture_coordinates` to `true` in your `embrace-config.json` to enable this feature
+- Several public APIs are now implemented in Kotlin rather than Java. Generally this will not affect backwards
```

Comment:
I tried to add a comment, and I approved it twice, so consider it super approved 😂 

Are these new line jumps intentional?